packages/components/src/List/ListComposition/index.js

```diff
@@ -8,7 +8,8 @@ import VList from './VList';
 
 export default {
 	DisplayMode,
-	InfiniteScrollList,
+	LazyLoadingList,
+	InfiniteScrollList: LazyLoadingList,
```

Comment:
Following @jmfrancois advice, also export InfiniteScrollList to avoid a breaking change.